app/controller/pyproject.toml

```diff
@@ -23,23 +21,32 @@ click = "^8.0.0"
 chainsail-runner-rexfw = { path = "../../lib/runners/rexfw", optional = true, develop = true }
 grpcio = "^1.35.0"
 regex = "^2022.8.17"
-black = "22.3.0"
```

Comment:
We didn't need `black` as a regular dependency. It's already specified as a dev-dependency.